app/components/preprint-form-upload.js

```diff
@@ -10,8 +11,31 @@ export const State = Ember.Object.extend(Ember.Freezable, {
 }).create().freeze();
 
 const component = CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
+    session: Ember.inject.service(),
+    currentUser: Ember.inject.service(),
+    fileManager: Ember.inject.service(),
     state: State.START,
     uploadState: State.START,
+    selectedNode: null,
+    user: null,
+    userNodes: Ember.A([]),
+    init() {
+        this._super(...arguments);
+        if (this.get('session.isAuthenticated')) {
+            this._setCurrentUser();
+        }
+    },
+    _setCurrentUser() {
+        this.get('currentUser').load().then(user => this.set('user', user));
+    },
+    onGetCurrentUser: Ember.observer('user', function() {
+        const user = this.get('user');
+        if (user) {
+            loadAll(user, 'nodes', this.get('userNodes'));
+        } else {
+            this.set('userNodes', Ember.A());
```

Comment:
I would recommend minimizing the amount of work that an individual component does to interact with the store. 

In particular for this case, multiple sections of the form might want access to the information, and shouldn't have to do so separately... one alternative is to have the route own the data (eg via setupController), and pass these variables into the component.